kuma/users/stripe_utils.py

```diff
@@ -47,13 +47,14 @@ def retrieve_and_synchronize_subscription_info(user):
 
             subscription_info = {
                 "id": stripe_subscription_info.id,
-                "amount": stripe_subscription_info.plan.amount,
+                # Amount is in cents, so divide by 100
+                "amount": stripe_subscription_info.plan.amount / 100,
```

Comment:
Can we move this to the view layer instead? Sending floating-point numbers via JSON is bad because it's unpredictable. Integers are not. I'd rather we communicate in cents and if we prefer to display it in dollars, we make that a formatting thing in our React code.